css/style.css

```diff
@@ -1,55 +1,58 @@
-html {
-    font-size: 120%;
-    margin: 60px;
-  }
+body {
```

Comment:
Style either the body or a div element that wraps the entire contents of the page, rather than the html element.